support/client/lib/vwf/model/mil-sym.js

```diff
@@ -783,6 +796,10 @@ define( [ "module",
                     modifierActualName = msa[ modObj.modifier ];
                     break;
                     
+                case "ModifiersTG":
```

Comment:
It looks like this `switch` statement is a candidate for being factored out into its own function since it is repeated.